internals/generators/organization/index.js

```diff
@@ -77,6 +77,12 @@ module.exports = {
         return "The captive portal logout action url required";
       },
     },
+    {
+      type: "input",
+      name: "host",
```

Comment:
I would call it openwisp_radius_url rather than `host`, to make it more obvious.